hsds/basenode.py

```diff
@@ -61,7 +68,14 @@ def getHeadUrl(app):
     req_reg = head_url + "/register"
     log.info("register: {}".format(req_reg))
 
-    body = {"id": app["id"], "port": app["node_port"], "node_type": app["node_type"]}
+    log.warn("env:{}".format(os.environ))
```

Comment:
Why are we dumping os.environ as a warning?

Reply:
Should have been taken out before check-in, this was temporarily useful for seeing what useful things DCOS was given in the environment.